src/Microsoft.Health.Fhir.Core/Features/Resources/Bundle/BundleMetricsNotification.cs

```diff
@@ -6,16 +6,14 @@
 using System.Collections.Generic;
 using Microsoft.Health.Fhir.Core.Features.Metrics;
 using Microsoft.Health.Fhir.Core.Models;
-using Microsoft.Health.Fhir.ValueSets;
-using static Hl7.Fhir.Model.Bundle;
 
-namespace Microsoft.Health.Fhir.Api.Features.Resources.Bundle
+namespace Microsoft.Health.Fhir.Core.Features.Resources.Bundle
```

Comment:
To document my learnings: `BundleMetricsNotification` now lives in a new namespace, which requires us to explicitly indicate that resources are being referenced from `Core.Resource` and not `Core.Features.Resource`. No string resources have been moved - this PR clears up the ambiguity that comes with this namespace change.